templates/main.js

```diff
@@ -0,0 +1,82 @@
+// =================JS for Mobile Menu=========================
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+const menuX = document.querySelector('#menu-x');
+const navLink = document.querySelectorAll('.nav-link');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  menuX.classList.toggle('active');
+});
+
+navLink.forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
+
+// ========================JS for manupulating the DOM Elements=======================
+
+const featuredSpeakers = [
+  {
+    profilePhoto: '../images/oprah-photo.jpg',
```

Comment:
- [ ] Please the link to your `profilePhoto` is broken, see the screenshot below and kindly fix the issue.

![brokenImages](https://user-images.githubusercontent.com/62849872/176904578-317df96f-8187-4fd6-ac72-abe3a07fec54.png)

Reply:
@t-yanick How can I fix that... I have tried several ways since morning and it is not working out. On my local machine the pictures are displaying. So, I am failing to understand why when I take it live it fails to display the pictures. Any guidance on how to go about it will be highly appreciated